admin_guide/install/docker_registry.adoc

```diff
@@ -358,6 +358,117 @@ Digest: sha256:3662dd821983bc4326bee12caec61367e7fb6f6a3ee547cbaff98f77403cab55
 ----
 ====
 
+== Exposing the Registry
+
+If you wish to expose your internal registry externally it is recommended that you run a
+link:#securing-the-registry[secure registry].  In order to expose the registry you must
+first have link:deploy_router.html[installed a router].
+
+. link:#deploy-registry[Deploy the registry].
++
+. link:#securing-the-registry[Secure the registry]
++
+. link:deploy_router.html[Deploy a router]
++
+. Create your passthrough route with `oc create -f <filename>.json`.  The passthrough route will
+point to the registry service that you have created.
++
+====
+----
+{
+    "kind": "Route",
+    "apiVersion": "v1",
+    "metadata": {
+        "name": "registry",
+        "namespace": "default",
+        "labels": {
+            "docker-registry": "default"
+        }
+    },
+    "spec": {
+        "host": "<host>", <1>
+        "to": {
+            "kind": "Service",
+            "name": "docker-registry" <2>
+        },
+	"tls": {
+	    "termination": "passthrough" <3>
+	}
+    },
+    "status": {}
+}
```

Comment:
If it's all the same, can we convert this to YAML instead to align w/ most other examples?